src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj

```diff
@@ -50,6 +50,14 @@
     <PublishReadyToRun Condition="'$(RuntimeIdentifier)' != ''">true</PublishReadyToRun>
   </PropertyGroup>
 
+  <!-- When we are packing each RID, we set PackRuntimeIdentifier; by default this will also get passed to the builds of all ResolveProjectReferences
+       which causes a lot of duplicate building. This removes it. -->
+  <ItemDefinitionGroup>
+    <ProjectReference>
+      <GlobalPropertiesToRemove>PackRuntimeIdentifier</GlobalPropertiesToRemove>
```

Comment:
I don't really know what is going on here.  If you can verify that all the dependency dlls are R2R images in the pack output I'm good.